Common/Data/Custom/Quiver/QuiverCongress.cs

```diff
@@ -0,0 +1,185 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using NodaTime;
+using ProtoBuf;
+using static QuantConnect.StringExtensions;
+using QuantConnect.Util;
+using Newtonsoft.Json.Converters;
+
+namespace QuantConnect.Data.Custom.Quiver
+{
+    public enum TransactionValue
+    {
+        Purchase,
+        Sale
+    };
+
+    public enum HouseValue
```

Comment:
Could we rename this to another value? `Congress` maybe? I think it's more intuitive to access it via `trade.House == Congress.Senate` or something along those lines. Let's talk on slack to see what you think 👍 

Also, could we put the enum in its own file? It'll make calling the value easier when users import the `QuantConnect.Data.Custom.Quiver` namespace in C# and Python